src/js/background.js

```diff
@@ -480,6 +480,7 @@ Badger.prototype = {
       Migrations.migrateDntRecheckTimes2,
       Migrations.forgetMistakenlyBlockedDomains,
       Migrations.unblockIncorrectlyBlockedDomains,
+      Migrations.unblockEff,
```

Comment:
Also, this doesn't just unblock, it removes any `eff.org` domains from Privacy Badger's memory. More like `forgetEFF`.

Reply:
We should forget the other known DNT domains while we are at it. For example, #1397 is about `medium.com`.

Reply:
@ghostwords I'd prefer to do that in a separate PR. Can you make an issue?

Reply:
No, I think this change belongs here. This bug isn't exclusive to the EFF. #1397, the issue we claim to be fixing, is about `medium.com`.

Reply:
Why is the function still called "unblock"?

Reply:
Should we add a `console.log`, like we do in the other migrations?

Reply:
I'd like to keep this PR small in the interest of getting it merged before the release Monday. However I can add `medium.com` assuming it won't cause further issues.

Because it unblocks the domains, I'll rename it to "forget" or something.

I'll add a console.log.